mappings/net/minecraft/network/packet/s2c/play/GameJoinS2CPacket.mapping

```diff
@@ -4,18 +4,33 @@ CLASS net/minecraft/class_2678 net/minecraft/network/packet/s2c/play/GameJoinS2C
 	FIELD field_12280 playerEntityId I
 	FIELD field_12281 hardcore Z
 	FIELD field_12282 gameMode Lnet/minecraft/class_1934;
-	FIELD field_12284 dimensionTracker Lnet/minecraft/class_5318$class_5319;
+	FIELD field_12284 registryTracker Lnet/minecraft/class_5318$class_5319;
+		COMMENT Accessor to dimension types.
```

Comment:
```suggestion
		COMMENT Accessor to synced registries.
```

Reply:
For now this `RegistryTracker.Modifiable` is hardcoded (yes, final) to support dimension types only, so.